tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PathDependencyTestCase.java

```diff
@@ -117,7 +117,7 @@ public void testBalaPathCase1() throws BallerinaTestException {
     public void testBalaPathCase2() throws BallerinaTestException {
         Path caseResources = tempTestResources.resolve("case2");
         // Build bee module of TestProject1
-        String beeModuleBalaFileName = "bee-" + ProgramFileConstants.IMPLEMENTATION_VERSION + "-any-1.2.0"
+        String beeModuleBalaFileName = "bee-" + RepoUtils.getSpecVersion() + "-any-1.2.0"
```

Comment:
In the new implementation we do not have the spec version in bala file name

Reply:
Removed this